sdk/security/keyvault/azcertificates/autorest.md

```diff
@@ -119,7 +206,7 @@ directive:
   # (specifying models because others have "ID" fields whose values aren't cert IDs)
   - from: models.go
     where: $
-    transform: return $.replace(/(type (?:Deleted)?Certificate(?:Bundle|Item) struct \{(?:\s.+\s)+\sID \*)string/g, "$1ID")
+    transform: return $.replace(/(type (?:Deleted)?Certificate(?:\s|Properties\s)struct \{(?:\s.+\s)+\sID \*)string/g, "$1ID")
```

Comment:
This can be simpler because there will always be a space between the type name and `struct`:
```suggestion
    transform: return $.replace(/(type (?:Deleted)?Certificate(?:Properties)? struct \{(?:\s.+\s)+\sID \*)string/g, "$1ID")
```

Reply:
fixed in 361ff3f